rollbar/__init__.py

```diff
@@ -648,9 +648,11 @@ def _check_config():
         return False
 
     # make sure we have an access_token
-    if not SETTINGS.get('access_token'):
-        log.warning("pyrollbar: No access_token provided. Please configure by calling rollbar.init() with your access token.")
-        return False
+    handler = SETTINGS.get('handler')
+    if not handler == 'agent':
+        if not SETTINGS.get('access_token'):
+            log.warning("pyrollbar: No access_token provided. Please configure by calling rollbar.init() with your access token.")
+            return False
```

Comment:
I think it might be a bit easier to read if it looked something like:

```python
# skip access token check for the agent handler
if SETTINGS.get('handler') == 'agent':
    return True

# make sure we have an access_token
# <access token check>
```